charts/spire/Chart.yaml

```
@@ -27,7 +27,7 @@ description: |
           - --service-account-signing-key-file=/run/config/pki/sa.key
   ```
 type: application
-version: 0.7.1
+version: 0.7.2
```

```suggestion
version: 0.8.0
```

As this brings a new feature.